sdk/avs/Azure.ResourceManager.Avs/src/Generated/IscsiPathData.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.Avs.Models;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.Avs
+{
+    /// <summary>
+    /// A class representing the IscsiPath data model.
+    /// An iSCSI path resource
+    /// </summary>
+    public partial class IscsiPathData : ResourceData
```

Comment:
`Iscsi` triggers a cspell check error. Add it into the exception list: https://github.com/Azure/azure-sdk-for-net/blob/a70001fa0085f344daf9ef0e131f3f5d85fe7971/.vscode/cspell.json#L235